.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/3rdParty/babylon"]
+	path = test/3rdParty/babylon
+	url = https://github.com/babel/babylon.git
```

Comment:
@markelog we can make it so that if the directory is missing, the tests still pass. This way regular contributors don't need to grab the submodule